src/pages/studyView/studyPageHeader/ActionButtons.tsx

```diff
@@ -15,6 +15,7 @@ import {AppStore} from "../../../AppStore";
 import {serializeEvent} from "../../../shared/lib/tracking";
 
 export interface ActionButtonsProps {
+    loaded: boolean;
```

Comment:
lets make this loadingComplete

Reply:
Sure. Do you have any particular naming pattern you're following here? It makes no difference to me, so if there is one, I'm happy to follow it if you point me in the right direction.

Reply:
Well, the mobx promise status flags (isComplete, isPending, isError) would be a good place to start.  So, it's not strict, but I think including "complete" is good.  So, I say "loadingComplete"